QuestionBot/QuestionBot/Model/Record.cs

```diff
@@ -1,5 +1,19 @@
-﻿namespace QuestionBot.Model {
+﻿using System;
+
+namespace QuestionBot.Model {
 
     public class Record : IRecord {
+        public string Question { get; set; }
+        public string Answer { get; set; }
+        public int ID { get; set; }
+        public DateTime TimeAsked { get; set; }
+        public DateTime TimeAnswered { get; set; }
+
+        public Record(int id, string qstn, DateTime qstnTime)
+        {
+            Question = qstn;
+            TimeAsked = qstnTime;
+            ID = id;
```

Comment:
super minor: it is a bit easier to read when these are in the same order as the parameters (assign id, then question, then timeasked)